src/internal/EnhancedButton.js

```diff
@@ -43,10 +43,11 @@ class EnhancedButton extends Component {
       PropTypes.string,
       PropTypes.element,
     ]),
+    disabled: PropTypes.bool,
+    disabledColor: PropTypes.string,
     disableFocusRipple: PropTypes.bool,
```

Comment:
[circle-ci](https://travis-ci.org/callemall/material-ui/builds/202065054) saying lines 48-50 are not in alphabetical order, but this doesn't seem to be the case.